packages/forms-web-app/src/views/selected-appeal/ip-comment-details.njk

```diff
@@ -31,7 +32,11 @@
 
 			<hr class="govuk-section-break govuk-section-break--visible govuk-section-break--m">
 
-			
-		</div>
-	</div>
+			{% for comment in appeal.formattedComments %}
+                <h2 class="govuk-heading-l">{{ comment.key.text }}</h2>
+                {{renderComment(comment.value, loop.index, "Comments") }}
+            {% endfor %}
+        </div>
+    </div>
+    <script src="/public/javascripts/commentActions.bundle.js"></script>
```

Comment:
this should probably be
{% block pageScripts %}
    <script src="/public/javascripts/commentActions.bundle.js"></script>
{% endblock %}

or use defer

Reply:
buttons stopped working using the block so went with defer

Reply:
it would have had to have been outside the block it's already in, was using locally so deffo works, this is good though

Reply:
Ah ok, good to know thank you!